cluster/saltbase/install.sh

```diff
@@ -24,6 +24,12 @@ set -o pipefail
 SALT_ROOT=$(dirname "${BASH_SOURCE}")
 readonly SALT_ROOT
 
+readonly DOCKER_IMAGES_FOR_MASTER=(
```

Comment:
yes, install.sh does not include any other scripts. so I cant easily share the array. I have another copy here.